src/core/client/stream/test/comments/permalink/permalinkViewRejectedComment.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { screen } from "@testing-library/react";
+import customRenderAppWithContext from "coral-stream/test/customRenderAppWithContext";
```

Comment:
Can you please update the imports in this file to be consistent with how we're importing elsewhere? (So, space before this import from "coral-stream/..." and then the imports from "./create" and "../../fixtures" below together with no space between)

Reply:
https://github.com/coralproject/talk/blob/fix/CORL-2261-children-of-rejected-comments/src/core/client/stream/test/comments/permalink/permalinkViewRejectedComment.spec.tsx#L5